tutorials/migration_plugins.md

```diff
@@ -5,18 +5,18 @@
 Gazebo Classic supports
 [6 different C++ plugin types](http://gazebosim.org/tutorials?tut=plugins_hello_world&cat=write_plugin),
 each providing access to different parts of the API, like physics, rendering,
-sensors, GUI, etc. Due to Ignition Gazebo's architecture based on an
+sensors, GUI, etc. Due to Gazebo's architecture based on an
```

Comment:
A lot of these `Gazebo` instances only make sense as `Gazebo Sim`

Reply:
https://github.com/gazebosim/gz-sim/pull/1496/commits/666b36b7d49bb19c50cc2c449032f28faea15082